mod.conf

```diff
@@ -1,3 +1,3 @@
 name = abriflame
-depends = default, fire, abriglass
+depends = fire, abriglass
 description = Adds coloured fire through use of fire starter tool.
```

Comment:
Note to fix in another PR: missing an endline.


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Inefficient